backend/app/core/middleware/logging.py

```
@@ -0,0 +1,109 @@
+
+Project: Kryptt
+Author: Jon
+Social Media:
+- Twitter: @jondoescoding
+"""
+
+import logging
+import json
+import time
+from typing import Callable
+from fastapi import FastAPI, Request, Response
```

Ruff (0.8.2)

`fastapi.FastAPI` imported but unused

Remove unused import: `fastapi.FastAPI`

(F401)